network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/author-images.html

```diff
@@ -2,9 +2,10 @@
 
 <div class="author-images d-flex flex-wrap">
     {% for author in authors %}
-        <div class="author-image d-flex align-items-center justify-content-center {% if author.image is None %}no-avatar{% endif %}" aria-label="{{ author.name }}">
-            {% if author.image %}
-            <img src="{% image_url author.image "fill-100x100" %}" alt="{{ author.name }}" title="{{ author.name }}" />
+        <div class="author-image d-flex align-items-center justify-content-center {% if author.author.image is None %}no-avatar{% endif %}" aria-label="{{ author.author.name }}">
```

Comment:
Let's clean this up just a tiny bit:

```
    {% for entry in authors %}
    {% with author=entry.author %}
```
and then we can refer to things like `author.image` rather than the `author.author.image` etc.

Reply:
Great suggestion Pomax! I have added it in the latest push and re-requested review. Thanks!